app/routes/pages/components/Sidebar.css

```diff
@@ -1,108 +1,49 @@
 @import url('~app/styles/variables.css');
 
-.side {
-  height: 100%;
-  display: flex;
-  justify-content: center;
-  box-sizing: border-box;
-  background-color: rgba(var(--rgb-min), var(--rgb-min), var(--rgb-min), 2%);
-  margin-bottom: 5em;
-  width: 290px;
-}
-
 .sidebar {
-  margin: 0 1.1em;
-  width: 100%;
-  padding-top: 20px;
   display: flex;
   flex-direction: column;
-
-  @media (--mobile-device) {
-    margin: 0;
-    padding-right: 1.5rem;
-  }
+  width: 290px;
+  flex-shrink: 0;
+  padding: 1.1em;
+  padding-bottom: 4.4em;
+  background-color: rgba(var(--rgb-min), var(--rgb-min), var(--rgb-min), 2%);
+  border-radius: var(--border-radius-lg) 0 0 var(--border-radius-lg);
+  z-index: 11;
 }
 
 .sidebarHeader {
-  margin-top: 20px;
-  width: 95%;
+  margin: 1em 0 1.4em;
+  align-self: center;
   text-align: center;
-  border-bottom: 2px solid var(--border-gray);
   padding-bottom: 0.6em;
+  width: 93%;
+  border-bottom: 2px solid var(--border-gray);
 }
 
-.sidebarPicture {
-  margin-top: 30px;
-  display: none;
-  flex-wrap: wrap;
-  align-items: center;
-  justify-content: center;
-}
-
-.sidebarTop {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-}
-
-.sidebarBottom {
-  margin-top: 2em;
-}
-
-.oldImg {
-  filter: grayscale(100%);
-}
-
-.pictureInfo {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-}
-
-.sidebarCloseBtn {
-  width: 40px;
-  height: 25px;
+.sidebarCloseButton {
   display: none;
-  align-items: right;
-  justify-content: flex-end;
-  color: var(--lego-font-color);
-  z-index: 10;
-
-  @media (--mobile-device) {
-    display: inline-block;
-    position: absolute;
-    right: 0.7em;
-    top: 0.5em;
-  }
 }
 
 @media (--mobile-device) {
-  .side,
-  .sidebarPicture {
-    display: none;
-  }
-
-  .side.isOpen {
-    display: block;
-    position: absolute;
-    z-index: 11;
-    width: 100%;
-    box-shadow: var(--shadow-lg);
-    height: 100%;
-  }
-
   .sidebar {
+    position: fixed;
+    top: 0;
     width: 100%;
-    max-width: 100%;
+    height: 100vh;
+    border-radius: 0;
     background-color: var(--lego-card-color);
-    height: 100%;
+    overflow-y: auto;
+    overscroll-behavior: none;
   }
 
-  .sidebarWrapper {
-    width: 100%;
-    position: absolute;
-    height: 100%;
-    background-color: rgba(var(--rgb-min), var(--rgb-min), var(--rgb-min), 7%);
-    z-index: 10;
+  .sidebarClosed {
+    display: none;
+  }
+
+  .sidebarCloseButton {
+    display: block;
+    align-self: flex-end;
```

Comment:
[`align-self` should in theory not apply to block-level boxes](https://developer.mozilla.org/en-US/docs/Web/CSS/align-self) - so these two lines sort of contradict each other.

Reply:
That makes sense, but it works for some weird reason. I need to set the display to something (in order to override the `display: none`) though. What should I set it to? `display: flex`?